lib/money/money.rb

```diff
@@ -76,7 +86,7 @@ def with_currency(new_currency)
 
     def default_settings
       self.parser = MoneyParser
-      self.default_currency = Money::NULL_CURRENCY
+      self.default_currency = opt_in_v1? ? nil : Money::NULL_CURRENCY
```

Comment:
I've done some digging for https://github.com/Shopify/money/pull/225 and nobody is setting it as far as I can tell - I propose we rip it out altogether.

Reply:
people may use this outside of shopify

Reply:
They can use the Rubocop rule and get most if not all instances fixed.